sockeye/lexical_constraints.py

```diff
@@ -25,6 +25,104 @@
 RawConstraintList = List[List[int]]
 
 
+class AvoidPhrase:
+    """
+    Represents a negative phrasal constraint. Tracks progress through a phrase
+    and returns a word ID if that word ID would complete the phrase.
```

Comment:
Returns the word ID in the `avoid()` method.

Reply:
Yes, clearer.